jobs/e2e_node/perf-image-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
```

Comment:
the existing job also runs for ubuntu, can we include that here?

Reply:
that is a good point... my decision making for this was purely based on sig node conversations about which images to run tests in and the cos ones are a definite yes.
Everything else that isn't cos seems to be up in the air at this moment (?).
During a previous meeting there was also a mention of organizing things such that each job only ran 1 image.

Right now, kubelet-flaky is running tests on this ubuntu image as well
https://github.com/kubernetes/test-infra/blob/dbd9fce65bd40199e01f6f5ee2e34ca51fa298b6/jobs/e2e_node/image-config.yaml#L6
so asking for feedback here: should this version of kubelet-flaky run ubuntu as well? and if so, should it run the same ubuntu image that it has been using until now or another?

/cc @spiffxp @MHBauer @bsdnet @vpickard

Reply:
@alejandrox1 Let's also update the Ubuntu since those are two images running. After we got them green, per the meeting, we may add more.
I will start a topic in sig-node to discuss how to support multiple images in a scalable way.

Reply:
I don't know who monitors the output of this (clearly nobody), nor do I know if it surfaces in a dashboard.

Reply:
Sweet, so im looking at all the ubuntu images for the ubuntu-os-gke-cloud project
```
gcloud compute images list --project ubuntu-os-gke-cloud
```

and `ubuntu-gke-1804-1-17-v20200605` seems to be the latest one.

As per monitoring this later on, we'll figure it out :sweat_smile: